cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -410,6 +418,11 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 	).Run()
 	time.Sleep(wait.Jitter(s.ControllerStartInterval.Duration, ControllerStartJitter))
 
+	// run the shared informers
+	for _, informer := range informers {
+		go informer.Run(wait.NeverStop)
```

Comment:
> Are shared informers ready for wider use, or are they still a work in progress?

They are ready for wider use.  The initial transitions of controllers to a shared pod informer went very smoothly, so I don't expect any structural problems.  Remember not to mess with the object in the cache, its shared.